### PR TITLE
Support for Azure DevOps repos

### PR DESCRIPTION
**Pull request by yradsmikham**
- Added changes made to build.sh that support Github and Azure DevOps repo
- Updated documentation on how to configure Flux with Azure DevOps repo

Addresses #88

### CODE REVIEW

File: gitops/azure-devops/build.sh [hunk 1]
```diff
@@ -1,8 +1,14 @@
-#!/bin/bash
-
-function copy_files() {
+function init() {
     cp -r * $HOME/
     cd $HOME
+
+    if [[ "$GIT_TYPE" == "github" ]]; then
```
Comment (andrebriggs) [inline gitops/azure-devops/build.sh]: @yradsmikham what is the result if not one of the handled conditions? What is $ACCESS_TOKEN is undefined. Perhaps we should have a fast fail in the beginning of the script if those variables are not defined or unexpected.

File: gitops/azure-devops/build.sh [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+        git_dest_repo="https://github.com/$AKS_MANIFEST_REPO"
+        git_type=$GIT_TYPE
+    elif [[ "$GIT_TYPE" == "azure" ]]; then
```
Comment (samiyaakhtar) [inline gitops/azure-devops/build.sh]: Is GIT_TYPE expected to be an environment variable? If so, maybe we should just add it to the env variables in documentation example [here](https://github.com/Microsoft/bedrock/blob/services/gitops/azure-devops/README.md)

Comment (andrebriggs) [reply gitops/azure-devops/build.sh]: It is in the new [documentation](https://github.com/Microsoft/bedrock/blob/89827537f9ae04be049f0acd2a56415f0e0defe2/gitops/azure-devops/README.md) that is part of the PR

Comment (samiyaakhtar) [reply gitops/azure-devops/build.sh]: Ok in that case it's a pipeline variable, not an env variable. That works. Resolving

File: gitops/azure-devops/build.sh (hunk 1 above)
Comment (yradsmikham) [reply gitops/azure-devops/build.sh]: Sure, I like that idea. Let me make that change and test it. Will get back to y’all.

**Pull request merged**